### DISCUSSION TITLE
Wireguard install failure

### DISCUSSION THREAD

**Discussion by Mysterox** [tteck/Proxmox#1197] [Help Needed] [answered]
Hi, i am trying to install wireguard with this script, but i always recieve this error message

Using Default Settings
**Using Container Type: Unprivileged NO DEVICE PASSTHROUGH**
Using Root Password: Automatic Login
Using Container ID: 115
Using Hostname: wireguard
Using Disk Size: 2GB
Allocated Cores 1
Allocated Ram 512
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Wireguard LXC using the above default settings
 - Validating Storage...bash: line 42: pvesm: command not found

[ERROR] in line 42: exit code 127: while executing command VALIDCT=$(pvesm status -content rootdir | awk 'NR>1')

What am i doing wrong?

**Comment by tteck** [comment]
pvesm - Proxmox VE Storage Manager 
How and where are you executing the Wireguard command?

**Reply by tteck** [reply]
Are you executing the command in the Proxmox shell or via a SSH terminal?

**Reply by Mysterox** [reply]
Via SSH Terminal, but with root user

**Reply by Mysterox** [reply]
Running directly from proxmox shell work fine!

Thanks for you quick reply!!!

**Comment by tteck** [comment, marked as answer]
**Run scripts within the Proxmox shell directly instead of using an SSH terminal.**

**Reply by tteck** [reply]
The reason why commands may not work properly when executed via SSH is that there could be differences in the environment variables and paths compared to when the same command is run directly within the Proxmox shell.
